scripts/ci/create-docker-image.sh

```diff
@@ -16,73 +19,90 @@ VERSION=${TAG#v}
 
 [ -n "$VERSION" ] && DOCKER_TAG=$VERSION || DOCKER_TAG=latest
 
+sha1=$( git rev-parse --short=12 HEAD )
+: ${SHA1:=$sha1}
+$WITH_SHA1 && [ "$DOCKER_TAG" != latest ] && : ${DOCKER_TAG:+-$SHA1}
+
 # See if a server forms part of DOCKER_IMAGE, e.g. DOCKER_IMAGE=registry.ng.bluemix.net:8080/skydive/skydive
 if [ "${DOCKER_IMAGE%/*/*}" != "${DOCKER_IMAGE}" ]; then
     DOCKER_SERVER=${DOCKER_IMAGE%/*/*}
 fi
 
-if [ -n "$PUSH_RUN" ]; then
-    echo "Running in push run mode. Skipping build."
-else
+docker_build() {
     for arch in $ARCHES
     do
+        flags="DOCKER_IMAGE=${DOCKER_IMAGE} DOCKER_TAG=${DOCKER_TAG}-${arch}"
```

Comment:
good - a trailing arch does make sense. Thanks for fixing this beginner uglyness of mine..

Reply:
k